src/check-primes.js

```diff
@@ -0,0 +1,20 @@
+function checkPrimeNumber(n) {
```

Comment:
isPrimeNumber
function return true or false should begin with these prefixes: should, has, is, did